src/wazuh_modules/agent_upgrade/wm_agent_upgrade_result_command.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Wazuh Module for Agent Upgrading
+ * Copyright (C) 2015-2020, Wazuh Inc.
+ * July 3, 2020.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+#include "wm_agent_upgrade.h"
+#include "wm_agent_parsing.h"
+#include "wm_agent_upgrade_tasks.h"
+#include "wazuh_db/wdb.h"
+
+cJSON* wm_agent_process_upgrade_result_command(const cJSON* agents) {
+    cJSON* response = cJSON_CreateArray();
+    cJSON* agents_info = wdb_select_agents_version(agents);
```

Comment:
Check if `agents_info` is not NULL and is an array.

Reply:
Outdaded... this function has been removed